### PR TITLE
[sp] fix suggestions overflow

### PR DESCRIPTION
**Pull request by shrey-mage**
# Summary
- extremely long column name links now wrap instead of overflowing `SuggestionRow`

# Tests

There's still an issue with "Remove outliers" because it needs to display the super long column name variable, which I'm not sure if we can do much about. It should be a very rare edge case.

image

cc: @tommydangerous @johnson-mage

### CODE REVIEW

File: mage_ai/frontend/oracle/elements/Link/index.tsx [hunk 1]
```diff
@@ -202,15 +203,16 @@ export const SHARED_LINK_STYLES = css<any>`
     text-overflow: ${props.textOverflow};
   `}
 
-  ${props => props.width && `
+  ${props => (props.minWidth || props.maxWidth) && `
```
Comment (johnson-mage) [inline mage_ai/frontend/oracle/elements/Link/index.tsx:206]: Are these props needed? `SHARED_TEXT_STYLES` already has minWidth and maxWidth props.

Comment (johnson-mage) [reply mage_ai/frontend/oracle/elements/Link/index.tsx:206]: And `LinkStyle` contains `SHARED_TEXT_STYLES`.

Comment (johnson-mage) [reply mage_ai/frontend/oracle/elements/Link/index.tsx:206]: The only difference is the existing props in `mage-ai/mage_ai/frontend/oracle/elements/Text/index.tsx` does not contain `display: block`, so maybe whatever Link component had the issue needs a `block` prop.

Comment (shrey-mage) [reply mage_ai/frontend/oracle/elements/Link/index.tsx:206]: oh I didn’t realize that it uses shared styles, the only thing is that Link needs `display: block` like you said

Comment (shrey-mage) [reply mage_ai/frontend/oracle/elements/Link/index.tsx:206]: anyways we can’t use `display: block` for `SuggestionsRow` bc it breaks the design of inline links, so it was more of a fix for the future. I’ll remove and only add `display: block`

Comment (johnson-mage) [reply mage_ai/frontend/oracle/elements/Link/index.tsx:206]: We have a `block` prop in `Link`, so check that.

**Review by johnson-mage — APPROVED** [review]
approved after removing redundant props
